### ISSUE TITLE
Wrong boot configuration?

### ISSUE THREAD

**Issue by lispyclouds** [liquidz/vim-iced#195]
Hello!

Firstly, thanks for the awesome work! 😄 

I followed the install [guide](https://liquidz.github.io/vim-iced/#installation) and was trying to fire up the `iced repl` on my boot [project](https://github.com/bob-cd/bob). But I get this error instead:

```
$ iced repl
OK: Boot project is detected
                                              boot.App.main                     App.java:  500
                                           boot.App.runBoot                     App.java:  407
org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke  ClojureRuntimeShimImpl.java:  145
org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke  ClojureRuntimeShimImpl.java:  154
                                                        ...
                                            boot.main/-main                     main.clj:  216
                                         boot.main/-main/fn                     main.clj:  216
                                   clojure.core/load-string                     core.clj: 4089
                                   clojure.core/load-reader                     core.clj: 4087
                                                        ...
                                    java.lang.Class.forName                   Class.java:  427
                                   java.lang.Class.forName0                    Class.java
                            java.lang.ClassLoader.loadClass             ClassLoader.java:  522
                                                        ...
                            java.lang.ClassLoader.loadClass             ClassLoader.java:  589
                                                        ...
                          java.net.URLClassLoader.findClass          URLClassLoader.java:  435
java.lang.ClassNotFoundException: cider.tasks
      clojure.lang.ExceptionInfo: cider.tasks
    line: 4
```
I looked up cider.tasks and found its a part of cider-nrepl. I added it as a dep in my `~/.boot/profile.boot` but still get this error.
What am I missing?

**Comment by lispyclouds** [comment]
Also if this helps:
```
$ java -version                                                                                                                                               
openjdk version "14" 2020-03-17
OpenJDK Runtime Environment (build 14+36-1461)
OpenJDK 64-Bit Server VM (build 14+36-1461, mixed mode, sharing)
```

**Comment by liquidz** [comment]
@lispyclouds Thanks for your reporting!
I could reproduce the problem. I'll fix it.

**Comment by liquidz** [comment]
@lispyclouds Could you try `dev` branch which I fixed?

**Comment by lispyclouds** [comment]
Yesss! this works! Awesome! Thanks a lot for the quick fix! 🎉 
```
$ iced repl
OK: Boot project is detected
REPL-y 0.4.3, nREPL 0.7.0
Clojure 1.10.1
OpenJDK 64-Bit Server VM 14+36-1461
        Exit: Control+D or (exit) or (quit)
    Commands: (user/help)
        Docs: (doc function-name-here)
              (find-doc "part-of-name-here")
Find by Name: (find-name "part-of-name-here")
      Source: (source function-name-here)
     Javadoc: (javadoc java-object-or-class-here)
    Examples from clojuredocs.org: [clojuredocs or cdoc]
              (user/clojuredocs name-here)
              (user/clojuredocs "ns-here" "name-here")
boot.user=>
```

**Comment by liquidz** [comment]
@lispyclouds Thanks!
I'll cut a patch release soon.

**Comment by liquidz** [comment]
Closed by #196
